text/0000-rfc-stages.md

```diff
@@ -0,0 +1,189 @@
+    - Start Date: (fill me in with today's date, YYYY-MM-DD)
+    - Relevant Team(s): (fill this in with the [team(s)](README.md#relevant-teams) to which this RFC applies)
+    - RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+    - Tracking: (URLs to project boards, quest issues, etc. Separate by a comma and space.)
+
+### Reconciling past RFCs
+
+For codebase-related RFCs that have already been merged, the release version is only required to be added to RFCs whose implementation was released after February 1st, 2018 (near the release of Ember 3). This is to preserve the time and effort of our contributors and volunteers.
+
+A stage will be applied to all previously merged RFCs.
+
+### Core team sign-off for Release
+
+The Core Teams' review of readiness for release has historically been an informal process. This RFC formalizes the responsibility by creating a stage for gathering confirmation that all necessary work is complete.
+
+When the work described in an RFC is ready for Release, a PR to make the change to the RFC is added by the RFC champion, and each team is requested as a reviewer. A representative of each team adds a review, and the community is also invited to comment via one week of FCP (final comment period). If a team does not respond to the request, it is assumed that the release may proceed. 
+
+Approvals for RFCs whose work is spread across multiple releases and packages are handled on a case-by-case basis. If the work for an RFC is spread across multiple releases of Ember, the RFC is considered to be in the Released stage when all features are complete.
+
+## How we teach this
+
+- The Stages section above will be added to the README of the RFCs repository.
+- Frontmatter will be added to the template for new RFCs (see below)
+- A blog post will announce the updates to the RFC process
+- Past RFCs should be updated to include their stage and release version, following the description in "Reconciling past RFCs."
+
+### Stages as communication tool
+
+Additionally, this RFC would unlock our ability to create a status board where everyone can easily see the progress of RFC. Today, if you need to know when an RFC feature was available in a stable release, you need to comb through release blog posts. Instead, we could use the frontmatter as data that powers the app. This app is not a requirement for this RFC, but rather an example of how we could use the information provided by stages.
+
+## Drawbacks
+
+Making updates to past RFCs can generate a lot of  GitHub notifications for people who watch the repository. We think that the benefit of knowing when an RFC is available in a stable release outweighs this drawback. Additionally, in the future, if an app displays the frontmatter data, someone could use that as their main source of information. Lastly, the Ember Times does a great job curating RFC updates, so someone could watch it instead if the RFC repo itself has too much information.
```

Comment:
I don't know if this is the place to leave comments about possible workarounds, but we could limit the change of past RFCs to one single PR so that it only creates a single notification for people watching the repo. I guess it would be a tradeoff between the size of this PR and the number of notifications 🤔